kernels/test/op_glu_test.cpp

```diff
@@ -128,6 +128,20 @@ TEST_F(OpGluOutTest, AllInputDoubleOutputSupport) {
 #undef TEST_ENTRY
 }
 
+TEST_F(OpGluOutTest, AllInputHalfOutputSupport) {
```

Comment:
This isn't testing when the inputs are Half/Bfloat16

Reply:
whoops! fixed